docs/editor/artificial-intelligence.md

```diff
@@ -0,0 +1,90 @@
+---
+Order: 7
+Area: editor
+TOCTitle: Refactoring
+ContentId:
+PageTitle: Code smarter with artifical intelligence
+DateApproved: 1/27/2023
+MetaDescription: Code smarter with AI-powered suggestions from GitHub Copilot.
+---
+# Code smarter with artificial intelligence
+
+In VS Code, you can enhance your coding with artificial intelligence (AI), such as suggestions for lines of code or entire functions, fast documentation creation, and help creating code-related artifacts like tests.
+
+[GitHub Copilot](https://copilot.github.com/) is an AI-powered code completion tool that helps you write code faster and smarter. You can use the [Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) in VS Code to generate code, or to learn from the code it generates.
+
+## Prerequisites
+
+You'll use the [GitHub Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) to power your AI suggestions in VS Code.
+
+> <a class="install-extension-btn" href="vscode:extension/	GitHub.copilot">Install the Copilot extension</a>
```

Comment:
Love the fast install flow. The link seems to have spaces.

Reply:
Ha good catch! Removed spaces.